rn/release-information/release-notes-21-08-update2.adoc

```diff
@@ -0,0 +1,80 @@
+
+// #30727 (both editions)
+* Adds support for OpenShift 4.9.0.
+
+// #32789, #32592 (both editions)
+* [WAAS] Adds support for an SNI-based proxy, so that you can serve multiple top-level domains, each with different TLS certificates, on the same internal port.
+Previously, when defining two apps using the same port with TLS, where each app had its own certificates, WAAS always used the certificate of the first app for both applications.
+As a result, the first app would return 200 OK, while the second app would be blocked by a certificate verification (TLS handshake) error.
+
+// #33257 (both editions)
+* Adds a debug mode for App-Embedded Defender, where all logs are also written to stdout.
+This makes it easier to debug crashes or other issues when Defender can't connect to Console.
+To enable the debug mode, specify an environment variable named `TW_DEBUG` and set it to `TRUE`.
+
+// #31630 (both editions)
+* Updates how serverless scan results are shown.
+Previously, if an in-progress serverless scan was cancelled, the results from the last scan were deleted, which could lead to empty or partial results being shown.
+Starting in 21.08 update 2, the last serverless scan results are retained if a currently running scan is cancelled by a newly invoked scan.
++
+NOTE: The new scan that caused the previous scan to be cancelled won't start or run.
+
+// #33779 (both editions)
+* Fixes how the *Manage > System > Scan > Unactionable vulnerabilities* toggle works, making it consistent with the UI text.
+Previously, when the toggle was enabled, it filtered out vulnerabilities by severity level only.
+Starting with 21.08 update 2, when the toggle is enabled, vulnerabilities are filtered by both severity level and fix plan.
+All vulnerabilities with a severity level of "negligible" and a fix plan of "won't fix" are filtered out.
+Vulnerabilities with a fix plan of "won't fix" are filtered regardless of severity.
+
+// #33519 (both editions)
+* Fixes a high memory usage issue that could cause Console to enter an infinite restart loop.
+
+// #33004 (both editions)
+* Adds the CAP_IPC_LOCK kernel capability to the Defender DaemonSet so that Defender can run on systems with insufficient mlock limits.
+
+// #32889 (both editions)
+* Adds coordinates for the Azure West US 3 region so that it's properly mapped to Arizona on the cloud radar (*Cloud > Radar*)
+
+// #32739 (both editions)
+* Adds missing /util endpoints to the OpenAPI spec file.
+
+// #32715 (both editions)
+* Fixes an issue where Defender containers failed liveness and readiness checks when a Defender DaemonSet was first deployed to an EKS cluster.
+Fixes an issue where Defender tries fetch host VM tags from AWS, while Defender is initializing and host network monitoring  is (*Radar > Settings*) is enabled.
```

Comment:
Is something missing in this description? I am trying to fill in the blanks...Was the issue seen when users enable host monitoring  and  defender is being initialized?
And should this read "Fixes an issue where Defender could not fetch host VM tags from AWS....

Reply:
Fixing this for clarity